lib/response.js

```diff
@@ -53,6 +55,7 @@ internals.Response.prototype.writeHead = function () {
 
 internals.Response.prototype.write = function (data, encoding) {
 
+    this.shot.payloadChunks.push(new Buffer(data, encoding));
```

Comment:
What do you think of https://github.com/hapijs/shot/pull/79/commits/7e66b7614882188ceb817f5e75fe770542007267 ?